src/test/java/ua/edu/internship/interview/service/business/SkillServiceIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+package ua.edu.internship.interview.service.business;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MongoDBContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+import ua.edu.internship.interview.data.repository.SkillRepository;
+import ua.edu.internship.interview.service.dto.skill.SkillTreeDto;
+import ua.edu.internship.interview.service.utils.exceptions.NoSuchEntityException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import java.util.List;
+
+@SpringBootTest
+@Testcontainers
+public class SkillServiceIntegrationTest {
+    @Container
+    public static MongoDBContainer mongoDBContainer = new MongoDBContainer(DockerImageName.parse("mongo:7.0.0"));
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.VisibilityModifierCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Variable 'mongoDBContainer' must be private and have accessor methods.
<!-- __reviewdog__:ChAxMDY4NzJkYTgyZTk2Y2E2EgpjaGVja3N0eWxl -->